app/controllers/api/v1/activity_sessions_controller.rb

```diff
@@ -104,31 +104,29 @@ def concept_tag_result_allowed_keys
   # Alias the following request parameters:
   # Map each result to the following structure:
   # {
-  #   concept_tag: "Creative Writing",
-  #   concept_class: "Writing Concepts",
+  #   concept: "uid-abcde12345",
   #   student_input: "The dog jumped over the cat."
   # },
   # Becomes this:
   # {
   #   metadata: {
-  #     concept_tag: "Creative Writing",
-  #     concept_class: "Writing Concepts",
+  #     concept: "uid-abcde12345",
   #     student_input: "The dog jumped over the cat."
   #   }
   # }
   #
-  # concept_tag_results -> concept_tag_results_attributes
+  # concept_results -> concept_results_attributes
   def transform_incoming_request
```

Comment:
could we instead change the api request made by the third party apps so that the metadata comes nested as the value of a metadata key, so that this transformation doesn't need to happen? Also I'm confused by the comment wherein the key/value pair for 'concept' becomes transformed so that its the value of the 'metadata' key, since in the #activity_session_params method it seems that 'concept' and 'metadata' are sibling keys ?